tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -634,15 +634,15 @@ public static VerifiableConsumer createFromArgs(ArgumentParser parser, String[]
             }
         }
 
-        String groupProtocol = res.getString("groupProtocol");
+        GroupProtocol groupProtocol = GroupProtocol.of(res.getString("groupProtocol"));
+        consumerProps.put(ConsumerConfig.GROUP_PROTOCOL_CONFIG, groupProtocol.name());
```

Comment:
Reviewer note: this change ensures the specified group protocol is set in the consumer configuration _regardless_ of its value. Before, the group protocol was only set in the configuration when it was `CONSUMER`.

This minor issue came out of the testing of switching the group protocol default from `CLASSIC` to `CONSUMER`.